generator/relation_tags.cpp

```diff
@@ -103,6 +103,7 @@ void RelationTagsWay::Process(RelationElement const & e)
   bool const processAssociatedStreet = type == "associatedStreet" &&
                                        Base::IsKeyTagExists("addr:housenumber") &&
                                        !Base::IsKeyTagExists("addr:street");
+  bool const isAssociatedStreet = type == "associatedStreet";
```

Comment:
```suggestion
  bool const isAssociatedStreet = type == "associatedStreet";
  bool const processAssociatedStreet = isAssociatedStreet &&
                                       Base::IsKeyTagExists("addr:housenumber") &&
                                       !Base::IsKeyTagExists("addr:street");

```
to avoid unnecessary string comparison.

Reply:
Fixed.